examples/goldilocks_models/initial_guess.cc

```diff
@@ -0,0 +1,195 @@
+//
+// Created by jianshu on 3/25/20.
+//
+#include "examples/goldilocks_models/initial_guess.h"
+
+namespace dairlib::goldilocks_models {
+//edited by Jianshu to try a new way of setting initial guess
+
+MatrixXd get_theta_scale(const string directory, int iter){
+//    VectorXd initial_theta = readCSV(directory + to_string(0) + string("_theta_s.csv"));
+    VectorXd current_theta = readCSV(directory + to_string(iter) + string("_theta_s.csv"));
+    VectorXd theta_scale = 1*VectorXd::Ones(current_theta.rows());
+    MatrixXd diag_theta_scale = theta_scale.asDiagonal();
+    return diag_theta_scale;
+}
+
+MatrixXd get_gamma_scale(int gamma_length, double min_sl, double max_sl, double min_gi, double max_gi,
+        double min_tr, double max_tr){
+    VectorXd gamma_scale(gamma_length);
+    gamma_scale << 1, 1, 2;
+    MatrixXd diag_gamma_scale = gamma_scale.asDiagonal();
+    return diag_gamma_scale;
+}
+
+string set_initial_guess(const string directory, int iter, int sample, int total_sample_num,
+                         double min_sl, double max_sl, double min_gi, double max_gi, double min_tr,
+                         double max_tr) {
```

Comment:
Just a heads up, I've been aware that there are functions in my codes, like this one, that have many many arguments, so I will probably create structs or classes to group some of the variables together.

Reply:
Good idea. I will modify them if you create a structure in the main block.